src/Blorc.OpenIdConnect.DemoApp/Program.cs

```diff
@@ -35,6 +35,10 @@
         options.RedirectUri = baseUrl;
         options.PostLogoutRedirectUri = baseUrl;
         options.AutomaticSilentRenew = true;
+        options.ResponseType = "code";
```

Comment:
In the AddBlorcOpenIdConnect method, I think we should check the options *after* calling this callback and throw an exception when responsetype != "code". This is because the documentation says this is the only thing we support?